dispatcher/src/test/java/io/camunda/zeebe/dispatcher/DispatcherTest.java

```diff
@@ -133,6 +133,34 @@ public void shouldNotClaimBeyondPublisherLimit() {
     verify(logBufferPartition0).getTailCounterVolatile();
   }
 
+  @Test
+  public void canClaimFragmentBatch() {
```

Comment:
:thought_balloon: I really dislike these tests, so I hope I can discuss with the reviewer a better approach.